magefile.go

```diff
@@ -3628,10 +3628,76 @@ func updateYamlFile(path string, keyVal ...struct {
 	return nil
 }
 
+// BuildDependencies builds the dependencies for the Elastic-Agent Helm chart.
 func (Helm) BuildDependencies() error {
 	return helm.BuildChartDependencies(helmChartPath)
 }
 
+// Package packages the Elastic-Agent Helm chart. Note that you need to set SNAPSHOT="false" to build a production-ready package.
+func (h Helm) Package() error {
+	mg.SerialDeps(h.BuildDependencies)
+
+	agentVersion := bversion.GetParsedAgentPackageVersion()
+	agentCoreVersion := agentVersion.CoreVersion()
+	agentImageTag := agentCoreVersion + "-SNAPSHOT"
```

Comment:
for some reason `SNAPSHOT=true` is not honoured

```
CHARTS_URL: "https://storage.googleapis.com/elastic-agent-helm-chart/elastic-agent-9.1.0-beta.tgz"
 ```

when using `SNAPSHOT=true mage helm:package`

is that the expected behaviour?

Reply:
`-beta` is the 'new' `snapshot`.